packages/core-components/src/components/Select/Select.tsx

```diff
@@ -101,21 +102,23 @@ const useStyles = makeStyles(
   { name: 'BackstageSelect' },
 );
 
-type Item = {
+export type SelectItem = {
   label: string;
   value: string | number;
 };
 
-type Selection = string | string[] | number | number[];
+export type Selection = string | string[] | number | number[];
```

Comment:
I wonder if this type is also a little too generic also,  sorry missed this last time also.

Let's go something like `SelectedItems`